src/algorithms/clustering/metrics.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import pandas as pd
+
+
+def distance(x, y):
+    return np.sqrt(np.sum(x**2 + y**2))
+
```

Comment:
shouldn't this be: `np.sqrt(np.sum((x-y)**2))` or better `np.linalg.norm(x-y)`

Reply:
Yes, It's true. I must have wrote this late.